ReportPortal.Tests/ReportPortal.E2E.UI.Tests/Tests/LaunchTests.cs

```diff
@@ -0,0 +1,124 @@
+﻿using FluentAssertions;
+using FluentAssertions.Execution;
+using NUnit.Framework;
+
+namespace ReportPortal.E2E.UI.Tests.Tests
+{
+    [TestFixture("Chrome")]
+    [TestFixture("Edge")]
+    public class LaunchTests : BaseTest
```

Comment:
Implement "5. User is able to move to appropriate launch view clicking on Launch name/total/passed/failed/skipped/to investigate places." per task 3

Reply:
Done